nsmdp/nsmdp.go

```diff
@@ -36,6 +36,7 @@ func NewNSMDevicePlugin() *NSMDevicePlugin {
 		DevicePlugin: deviceplugin.NewDevicePlugin(serverSock, resourceName),
 		devs:         make(map[string]*NSMDevice),
 		stop:         make(chan interface{}),
+		updatedevs:   make(chan *NSMDevice, 10),
```

Comment:
This is the change in #29, if we merge these two together, I'll just close #29. Or, I can remove this and we can merge them separately.